pkg/server/endpoints/harvester.go

```diff
@@ -88,28 +107,36 @@ func (e *Endpoints) postBundleHandler(ctx echo.Context) error {
 		return err
 	}
 
-	if harvesterReq.Bundle != nil && currentStoredBundle != nil && !bytes.Equal(harvesterReq.Bundle.Digest, currentStoredBundle.Digest) {
+	if harvesterReq.Bundle != nil &&
+		currentStoredBundle != nil &&
+		!bytes.Equal(harvesterReq.Bundle.Digest, currentStoredBundle.Digest) {
```

Comment:
You can simplify the if sentence to make it more readable.
```suggestion
        notNil :=  harvesterReq.Bundle != nil && currentStoredBundle != nil
        notEqual := !bytes.Equal(harvesterReq.Bundle.Digest, currentStoredBundle.Digest)
	if notNil && notEqual {
```